webgme/metadata.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
The import in `webgme-plugin-blob-tests.ts` requires the file be present.
    import MetaDataStr = require("text!metadata.json");